src/components/CovidData.js

```diff
@@ -0,0 +1,74 @@
+import React, { useEffect } from 'react';
+import { Link } from 'react-router-dom';
+
+import { useSelector, useDispatch } from 'react-redux';
+import { FaArrowCircleRight } from 'react-icons/fa';
+import { fetchData, getAllData } from '../redux/covid/covidSlice';
+import './CovidData.css';
+
+const CovidData = () => {
+  const dispatch = useDispatch();
+  const {
+    data, allData, loading, error,
+  } = useSelector((state) => state.covid);
+
+  useEffect(() => {
+    dispatch(fetchData());
+  }, [dispatch]);
+
+  useEffect(() => {
+    dispatch(getAllData());
+  }, [dispatch]);
+
+  if (loading) {
+    return <p>Loading...</p>;
+  }
+
+  if (error) {
+    return <p>{error}</p>;
+  }
+
+  return (
+    <main className="main">
+      <section className="All-data">
+        <h1>Covid 19 reports from 2020 to now</h1>
+        <div className="confirmed">
+          <p>
+            Cases:
+            <br />
+            {allData.cases}
+          </p>
+          <p>
+            Death(s):
+            <br />
+            {allData.deaths}
+          </p>
+        </div>
+      </section>
+      <section className="data-container">
+        {data.map((country) => (
+          <div key={country.country} className="covid-data">
+            <Link className="covid-data" to={`/covid/${country.countryInfo.iso2}`}>
+              <div className="data-wrapper">
+                <FaArrowCircleRight className="abs right-arrow" />
+                <img src={country.countryInfo.flag} alt={country.country} />
+                <p className="data name">
+                  {country.country}
+                </p>
+                <p className="data population">
+                  cases: (
+                  {' '}
+                  {country.cases}
+                  {' '}
+                  )
+                </p>
+              </div>
+            </Link>
+          </div>
+        ))}
+      </section>
```

Comment:
- [ ] You did an excellent job of organizing the data into cards :clap: . However, I believe we should stick to the original design. 
What I mean is that instead of having gaps between the cards, let's separate them by differentiating the background colour, as in the original design. :ok_hand:  😊

### your design
![Screenshot from 2023-04-25 15-18-56](https://user-images.githubusercontent.com/89228743/234329320-ad0729db-86c0-4094-b372-df7fec02567a.png)


### The original design
![Screenshot from 2023-04-25 15-27-36](https://user-images.githubusercontent.com/89228743/234329360-0c7c9499-c829-42b3-b41e-ecca8d19f3df.png)



- [ ] Instead of the one you used for the main bleu colour, I recommend using `#4369b2`. 

- [ OPTIONAL ] You can use colour pickers to get the correct colour from the template because we can't copy them. I can recommend a chrome extension called [Eye Dropper](https://chrome.google.com/webstore/detail/eye-dropper/hmdcmlfkchdmnmnmheododdhjedfccka?hl=en).

Reply:
Thank you @ridwanediallo , I applied your suggestions